clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1169,7 +1169,7 @@ def create_file(image, modality, total, bids_dir, mod_to_update):
             zip_image = "y"
 
         if image.Is_Dicom:
-            command = f"dcm2niix -b {generate_json} -z {zip_image} -o {output_path} -f {output_filename} {image_path}"
+            command = f"dcm2niix -w 0 -b {generate_json} -z {zip_image} -o {output_path} -f {output_filename} {image_path}"
```

Comment:
It seems that the `-w` option was introduced in [version v1.0.20190902](https://github.com/rordenlab/dcm2niix/releases/tag/v1.0.20190902). It's not mentioned in the releases notes but I with the old version of `dcm2niix` that I had in my laptop this doesn't work (I tested the released packages one by one until I found where it was introduced 😵 ) . We need probably to add a message in the documentation asking for a requirement  `dcm2niix >= v1.0.20190902` (not semantic) or just "a recent version of `dcm2niix`.

Reply:
> It seems that the `-w` option was introduced in version v1.0.20190902.

Very good point 👍 

> We need probably to add a message in the documentation asking for a requirement `dcm2niix >= v1.0.20190902`

Another good reason to use Conda to deploy `clinica`, since such versioned requirement can be specified.